cypress/features/visual/alert.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Alert component
+  I want to change Alert component properties
+
+  Background: Open Alert component page
+    Given I open "Alert" component page with button
+
+  @applitools
+  Scenario Outline: Change Alert component title to <title>
+    When I set title to "<title>"
+      And I open component preview
+    Then Element displays correctly
+    Examples:
+      | title                   |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <>                       |
+
+  @applitools
+  Scenario: Check open click event
+    When clear all actions in Actions Tab
+      And I open component preview
+    Then Element displays correctly
```

Comment:
Scenario should be:
Given clear all actions in Actions Tab (setting pre-requisites)
When I open component preview (Test point)
Then Element displays correctly (Result)

Reply:
I think the Feature section for these tests should specifically state these are visual tests to distinguish them from those in the /regression suite, e.g. "Feature: I want to check Alert component displays correctly when properties are changed". Similarly for Scenario Outline and Scenario.

Reply:
Agree, these are just demo tests which have been copied from regression

Reply:
We are going to change approach for visual tests, using existing ones rather than writing a whole new suite